libs/dh/market-participant/actors/feature-actors/src/lib/drawer/dh-actor-drawer.component.ts

```diff
@@ -92,7 +117,25 @@ export class DhActorDrawerComponent {
     query: GetActorByIdDocument,
   });
 
+  private getActorAuditLogByIdQuery$ = this.apollo.watchQuery({
+    errorPolicy: 'all',
+    returnPartialData: true,
+    useInitialLoading: true,
```

Comment:
suggestion: `returnPartialData` is probably not needed because the complete dataset is loaded here, there's no cached data for some of the fields.


```suggestion
    errorPolicy: 'all',
    useInitialLoading: true,
```